alvr/audio/src/lib.rs

```diff
@@ -30,11 +30,11 @@ use std::{
 
 static VIRTUAL_MICROPHONE_PAIRS: Lazy<HashMap<&str, &str>> = Lazy::new(|| {
     [
+        ("Line 1", "Line 1"),
         ("CABLE Input", "CABLE Output"),
         ("VoiceMeeter Input", "VoiceMeeter Output"),
         ("VoiceMeeter Aux Input", "VoiceMeeter Aux Output"),
         ("VoiceMeeter VAIO3 Input", "VoiceMeeter VAIO3 Output"),
-        ("Virtual Cable 1", "Virtual Cable 2"),
```

Comment:
There's definitely still enough installations floating around with the old device names since there aren't any auto updates for VAC. We should have something to match and handle both.